taxi/views.py

```diff
@@ -111,7 +167,7 @@ class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
 def toggle_assign_to_car(request, pk):
     driver = Driver.objects.get(id=request.user.id)
     if (
-        Car.objects.get(id=pk) in driver.cars.all()
+            Car.objects.get(id=pk) in driver.cars.all()
```

Comment:
The comment suggests checking if the car exists before accessing it. Consider adding error handling to manage cases where the car with the given `pk` does not exist, which would prevent potential exceptions.